### DISCUSSION TITLE
Off-site backup for recordings?

### DISCUSSION THREAD

**Discussion by epchris** [blakeblackshear/frigate#2689] [General]
I've got frigate running and producing events and recordings in a way that I like, I followed the guide to enable recordings but set recording retention to 0, then on "person" detection, it saves recordings for up to 10 days.

I'm putting my recordings in TrueNAS and was thinking I could just set up a task to sync the recordings directory to backblaze every 5 minutes.  What I'm seeing is about 50Mb of data transferred every 5 minutes, and what I think that means is that the recordings are happening all the time, and then certain ones related to "people" events are prevented from being "cleaned up" per the "0 days" recording retention...so I'm always getting churning recording files as some are kept around for a short bit before being cleaned up.

Does anyone do off-site backups of their event recordings, and if so how? 

Thanks!

**Comment by blakeblackshear** [comment]
Note that the 0.10.0 beta no longer writes continuously. Segments are only copied from cache when events occur.

**Reply by epchris** [reply]
oh, that's exactly what I need!  Any outstanding big issues with the beta?

**Reply by blakeblackshear** [reply]
No major issues. I just still have some improvements to make on stationary objects. I have been running it for weeks without issues. Just read the release notes.

**Reply by epchris** [reply]
Thanks! I appreciate all the work you've put into it
